kodilla-exception/src/main/java/com/kodilla/exception/test/FlightCheck.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class FlightCheck {
+
+    public void findFlight(Flight flight) throws RouteNotFoundException {
+        HashMap<String, Boolean> airports = new HashMap<>();
+        airports.put("Jasionka", true);
+        airports.put("Stansed", true);
+
+
+
+
+            if (airports.get(flight.getArrivalAirport())) {
+
+                System.out.println("Available");
+            } else {
+
+                System.out.println("Not");
+            }
+
+        airports.containsKey(flight);
```

Comment:
`airports.containsKey(flight);` zwraca booleana. Chodzi o to zeby wykorzystac tego booleana uzywajac np if'a i rzucac `RouteNotFoundException` kiedy elementu nie bedzie w mapie

Reply:
rzucenie wyjatkie powinno odbyc sie przed wyciagnieciem z mapy klucza. Przeciwnym wypadku bedzie dochodzic do wyjatku NullPointerException